std/io/bufio.ts

```diff
@@ -426,17 +426,6 @@ abstract class AbstractBufBase {
   buffered(): number {
     return this.usedBufferBytes;
   }
-
-  checkBytesWritten(numBytesWritten: number): void {
-    if (numBytesWritten < this.usedBufferBytes) {
-      if (numBytesWritten > 0) {
-        this.buf.copyWithin(0, numBytesWritten, this.usedBufferBytes);
-        this.usedBufferBytes -= numBytesWritten;
-      }
-      this.err = new Error("Short write");
```

Comment:
Don't know if this was done for a specific reasons, but I think `BufWriter.flush` should always write all bytes.